src/worldfactory.cpp

```diff
@@ -1060,6 +1078,10 @@ int worldfactory::show_worldgen_tab_modselection( const catacurses::window &win,
                 redraw_description = true;
                 redraw_headers = true;
             }
+        } else if( action == "VIEW_MOD_DESCRIPTION" ) {
+            if( const MOD_INFORMATION *selmod = get_selected_mod() ) {
+                popup( mman_ui->get_information( selmod ).c_str() );
```

Comment:
Please write this as `popup( "%s", get_information() )` - the current code uses the result of `get_information` as formatting string (contained '%' get interpreted).

Reply:
Thanks; good catch.  Fixed.